client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import FormCheckbox from 'components/forms/form-checkbox';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+import { isModuleActive, getModule } from 'state/jetpack-settings/modules/selectors';
+
+const MediaSettings = ( props ) => {
+	return (
+		<div>
+			<SectionHeader label={ props.translate( 'Media' ) }>
+				<Button
+					compact
+					primary
+					onClick={ props.submitFormAndActivateCustomContentModule }
+					disabled={ props.fetchingSettings || props.submittingForm }>
+					{ props.submittingForm ? props.translate( 'Saving…' ) : props.translate( 'Save Settings' ) }
+				</Button>
+			</SectionHeader>
+			<Card className="media-settings__card site-settings">
+				<FormFieldset>
+					<div className="media-settings__fieldset-infopopover">
+						<InfoPopover>
+							{ props.photonDescription }
+						</InfoPopover>
+					</div>
+					<JetpackModuleToggle
+						siteId={ props.site.ID }
+						moduleSlug="photon"
+						label={ props.translate( 'Speed up your images and photos with Photon.' ) }
+						description="Enabling Photon is required to use Tiled Galleries."
+						/>
+				</FormFieldset>
+				<FormFieldset className="has-divider is-top-only">
+					<div className="media-settings__fieldset-infopopover">
+						<InfoPopover>
+							{ props.carouselDescription }
+						</InfoPopover>
+					</div>
+					<JetpackModuleToggle
+						siteId={ props.site.ID }
+						moduleSlug="carousel"
+						label={ props.translate( 'Transform image galleries into full screen slideshows.' ) }
+						/>
+					{
+						( props.jetpackCarouselModuleActive && (
+								<div className="media-settings__module-settings is-indented">
+								<FormLabel>
+									<FormCheckbox
+										disabled={ props.submittingForm }
+										name="carousel_display_exif" />
+									<span>{ props.translate( 'Show photo metadata (Exif) in carousel, when available' ) }</span>
+								</FormLabel>
+								<FormLabel htmlFor="carousel_background_color">
```

Comment:
The styling on this one has me puzzled. The settings styles are pretty janky and instead of using specific classes, they rely on order of items which isn't ideal. This one in particular is being overridden by `label + label` in a selector. That means, the rule targets any label that is followed by another label.

We have three obvious solutions. One is messy (and maybe time-consuming), but ultimately will improve the quality and flexibility of the project. The other is a bit of a hack.

## Option one: A New Hope (or at least a new component)
We make a new component to be explicitly used to style the type of labels that should be bold. We then replace all the old labels that use this style with the new component and clean up the conflicting CSS.

## Option two: The Empire Strikes Back
We <del>build a Death Star</del> add an empty div between the labels. This seems particularly nasty in that it's not particularly intuitive to do when adding settings.

## Option three: Return of the Jedi
I modify the current FormLabel to accept custom classes or props and add a prop to add the bold styling.

What do you think?